File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.cs
using System;
using System.Linq;
public class Solution 
{
    public int[] solution(int[] emergency) 
    {
        int[] rankList = new int[emergency.Length];
        int count = 1;
        while(count != emergency.Length + 1)
        {            
            int max = emergency.Max();
            for(int i=0; i<emergency.Length; i++)
            {
                if(emergency[i] == max)
                {
                    rankList[i] = count;
                    emergency[i] = 0;
                    count++;
                }
            }
        }
        return rankList;
    }
}

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> emergency) 
{
    vector<int> answer(emergency.size(),1);
    
    for(int i=0; i<emergency.size(); i++)
    {
        for(int &c : emergency)
        {
            if(emergency[i] < c)
            {
                answer[i]++;
            }
        }
    }
    return answer;
}